{{cookiecutter.project_shortname}}/webpack.config.js

```diff
@@ -77,5 +81,37 @@ module.exports = (env, argv) => {
                 },
             ],
         },
+        optimization: {
+            minimizer: [
+                new TerserPlugin({
+                    sourceMap: true,
+                    parallel: true,
+                    cache: './.build_cache/terser',
+                    terserOptions: {
+                        warnings: false,
+                        ie8: false
+                    }
+                })
+            ],
+            splitChunks: {
+                name: true,
+                cacheGroups: {
+                    async: {
+                        chunks: 'async',
+                        minSize: 0,
+                        name(module, chunks, cacheGroupKey) {
+                            return `${cacheGroupKey}-${chunks[0].name}`;
+                        }
+                    }
```

Comment:
Adding the "shared" async chunk for code that repeats across multiple async chunks here would be good as this will have already solved the problem for component developers. 

https://github.com/plotly/dash-core-components/blob/dev/webpack.config.js#L124

Reply:
It would still require the developer to add the entry in `__init__.py` but maybe we can make a comment to that effect there or in the README instead of leaving them to fend off for themselves :)